messagebird/client.py

```diff
@@ -110,6 +111,16 @@ def message_delete(self, id):
         """Delete a message from the dashboard."""
         self.request_plain_text('messages/' + str(id), 'DELETE')
 
+    def mms_create(self, originator, recipients, body, mediaUrls, subject = None, reference = None, scheduledDatetime = None):
```

Comment:
I'm not sure where the documentation is or how explicit it may be, but there could be more info on the parameters of these methods I think. Now the user has to figure out by reading through the code what the exact format is for `scheduledDatetime`. Adding a small docstring here would help

Reply:
Added docstring to the function